clang/docs/StandardCPlusPlusModules.rst

```diff
@@ -1391,17 +1381,17 @@ with the following one:
   $ clang++ -std=c++20 -xc++-system-header --precompile iostream -o iostream.pcm
   $ clang++ -std=c++20 -fmodule-file=iostream.pcm --precompile M.cppm -o M.cpp
 
-In the latter example, the Clang could find the BMI for the ``<iostream>``
-so it would try to replace the ``#include <iostream>`` to ``import <iostream>;`` automatically.
+In the latter example, Clang can find the BMI for ``<iostream>`` and so it
+tries to replace the ``#include <iostream>`` with ``import <iostream>;``
+automatically.
 
 
 Relationships between Clang modules
 -----------------------------------
 
-Header units have pretty similar semantics with Clang modules.
-The semantics of both of them are like headers.
-
-In fact, we could even "mimic" the sytle of header units by Clang modules:
+Header units have similar semantics to Clang modules. The semantics of both are
+like headers. In fact, header units can be mimicked by Clang modules as in the
```

Comment:
```suggestion
like headers. Therefore, header units can be mimicked by Clang modules as in the
```

Reply:
Do we really mean 'Clang Modules' here?  I thought we were referring to C++ Standards Modules throughout here?

Reply:
I think we do mean Clang modules here, this is in a section about the relationship between C++ modules and Clang modules.